packages/frontend/src/components/Dataset/Dataset.scss

```diff
@@ -1,6 +1,11 @@
 .dataset {
   display: grid;
 
+  .data-link-title{
+    &:focus-visible{
+      outline:3px solid #000;
```

Comment:
Similar to the last comment, remember to stick to existing code conventions. In this case, adding a space in between the colon and the 3px